config/locales/zxcvbn/en.yml

```diff
@@ -0,0 +1,38 @@
+en:
+  zxcvbn:
+    feedback:
+      "Use a few words, avoid common phrases": Use a few words, avoid common phrases
+      "No need for symbols, digits, or uppercase letters": >-
+        No need for symbols, digits, or uppercase letters
+      "Add another word or two_ Uncommon words are better_": >-
```

Comment:
Personally I think it would be easier to just quote the values and keep them on one line rather than worrying about line breaks? But I don't feel that strongly:

```
"Add another word or two_ Uncommon words are better_": "Add another word or two. Uncommon words are better."
```